src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/headerbag.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends '@WebProfiler/Profiler/bag.html.twig' %}
+
+{% set labels = labels|default(['Header', 'Value']) %}
+
+{% block body %}
+    {% for key in bag.keys|sort %}
+        {% for value in bag.get(key, [], false) %}
```

Comment:
On the request panel, what about keeping one line by header but exposing the value as an array if needed instead?